advanced/image_processing/examples/plot_display_face.py

```diff
@@ -1,26 +1,26 @@
 """
-Display lena
-=============
+Display a Face
+==============
 
-An example that displays lena with matplotlib.
+An example that displays a face with matplotlib.
```

Comment:
> > +An example that displays a face with matplotlib.

> Racoon. Scipy calls it a racoon face.

Sounds good to me. I would write a "racoon face", rather than "face".